imperial.js

```diff
@@ -112,7 +114,13 @@ export default class Imperial {
   }
 
   tick(action) {
-    if (action.type === "playerSeating") {
+    if (action.type === "init") {
+      const { players, order, nations } = setup(action.payload);
+      this.players = players;
+      this.order = order;
+      this.nations = nations;
+      return;
```

Comment:
We don't want to run any side effects that may happen in the rest of `#tick`, so we exit early.